src/main/scala/org/hammerlab/guacamole/alignment/AffineGapPenaltyAlignment.scala

```diff
@@ -37,12 +37,12 @@ object AffineGapPenaltyAlignment {
         closeGapProbability
       )
 
-    val ((refStartIdx, path, score), refEndIdx) =
+    val ((refEndIdx, path, score), refStartIdx) =
```

Comment:
Somewhat, they weren't used anywhere, but when I added some tests and wanted to use them I noticed.